packages/@okta/vuepress-theme-prose/components/LiveWidget.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <div class="live-widget-wrapper"> 
+
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+          <h1 class="header">Instantly customize your login page</h1>
+          </div>
+        </div>
+      </div>
+      
+      <div class="background-curve"></div>
+      <div class="background-darkblue">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-6 offset-lg-6 col-xl-8 offset-xl-4">
+            <div class="button-holder">
+              <button class="toggle-button" :disabled="jsTabShown" v-on:click="toggleTabs" > JS </button>
+              <button class="toggle-button" :disabled="!jsTabShown" v-on:click="toggleTabs"> SCSS </button>
+            </div>
+          </div>
+        </div>
+        <div class="row widgets-wrapper">
+          <div class="col-sm-12 col-md-12 col-lg-6 col-xl-4">
+            <LiveWidgetOktaWidget :configJS="jsValid" :configSCSS="computedSCSS"/>
+          </div>
+          <div
+              class="col-sm-12 col-md-12 col-lg-6 col-xl-8"
+            >
+            <div class="cm-wrapper">
+              <LiveWidgetJSCodeMirror 
+                 v-if="jsTabShown"
+                :initialConfig="configJS" 
+                v-on:cmJSValueSet='onJSCodeChange' 
+              />
+            <LiveWidgetSCSSCodeMirror 
+               v-if="!jsTabShown"
+              :initialConfig="configSCSS"
+              v-on:cmCSSValueSet='onSCSSCodeChange' 
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      </div>
+  </div>
+</template>
+
+<script>
+  import {initialJSWidgetConf, initialWidgetSCSS} from '../const/live-widget.const'
+  import sass from '../util/sass/sass'
+  export default {
+    name: 'LiveWidget',
+    components: {
+      LiveWidgetSCSSCodeMirror: () => import('./LiveWidgetSCSSCodeMirror'),
+      LiveWidgetJSCodeMirror: ()=> import('./LiveWidgetJSCodeMirror'),
+      LiveWidgetOktaWidget: ()=> import('./LiveWidgetOktaWidget')
+    },
+    data() {
+      return {
+        jsTabShown: true,
+        configJS:  initialJSWidgetConf,
+        configSCSS: initialWidgetSCSS,
+        computedSCSS: '',
+        sassCompiler: {}
+      }
+    },
+    mounted: function(){
+      this.sassCompiler = new sass()
+      this.loadSassFiles()
+      this.sassCompiler.compile(this.configSCSS, (res)=> {
+          this.computedSCSS = res.text
+        })
+      },
+    computed: {
+      jsValid: function(){
+        return this.makeValidJSON(this.configJS)
+        },
+    }, 
+    methods: {
+      toggleTabs(){
+        this.jsTabShown = !this.jsTabShown
+      },
+      makeValidJSON(dirtyJSON){
```

Comment:
Why do we need to do 'fix' the JSON entered? Shouldn't we count on the user to write proper JSON?